extension-iac/src/iac_android.cpp

```diff
@@ -211,10 +128,10 @@ extern "C" {
 JNIEXPORT void JNICALL Java_com_defold_iac_IACJNI_onInvocation(JNIEnv* env, jobject, jstring jpayload, jstring jorigin)
 {
     IACCommand cmd;
-    cmd.m_Type = CMD_INVOKE;
-    cmd.m_Payload = StrDup(env, jpayload);
-    cmd.m_Origin = StrDup(env, jorigin);
-    QueueCommand(&cmd);
+    cmd.m_Command = IAC_INVOKE;
+    cmd.m_Payload = (void*)StrDup(env, jpayload);
+    cmd.m_Origin = (void*)StrDup(env, jorigin);
```

Comment:
Actually, I'd change the type in the struct to "const char*", and instead do the cast in "free((void*)cmd.m_Payload)" instead. That way the info is more debuggable in case it's needed

Reply:
Good point. Not sure why I did the other way around. Changed it.